components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1292,6 +1292,10 @@ void AdsServiceImpl::PurgeOrphanedAdEventsForType(
   }
 }
 
+void AdsServiceImpl::ClearData() {
+  LOG(ERROR) << "TO BE IMPLEMENTED";
```

Comment:
just checking. Is it planned to implement it in a follow up or it's missed?

Reply:
This is added in the PR https://github.com/brave/brave-core/pull/25050, which I'm waiting to be merged soon.
When the PR is merged then the current PR will be rebased.